var/spack/repos/builtin/packages/zpares/package.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Zpares(MakefilePackage):
+    """z-Pares is designed to compute a few eigenvalues
+       and eigenvectors of sparse matrices."""
+
+    homepage = "https://zpares.cs.tsukuba.ac.jp/"
+    url      = "http://zpares.cs.tsukuba.ac.jp/download/zpares_0.9.6a.tar.gz"
+
+    version('0.9.6a', sha256='3c34257d249451b0b984abc985e296ebb73ae5331025f1b8ea08d50301c7cf9a')
+
+    variant('mpi', default=False, description='Activates MPI support')
+    variant('mumps', default=False, description='Activates MUMPS support')
+
+    conflicts('+mumps', when='~mpi')
```

Comment:
Can we have a single variant with 3 values, instead of 2 variants and a conflict?

Reply:
Apologies for my late reply.
I found that mumps is available without MPI.
So I modified package,py. please confirm that.